.github/workflows/ci.yml

```diff
@@ -108,7 +108,7 @@ jobs:
           - ubuntu-latest
           - [self-hosted, linux, arm64]
           - macos-latest
```

Comment:
```suggestion
          - macos-12-large
```

Reply:
Shouldn't it be `macos-13-large`, actually?

Reply:
No, because then we stop supporting macos-12. `macos-latest` = `macos-12` we just want the bigger instance of `macos-12-large`.